### DISCUSSION TITLE
A way to change manifestFilePath?

### DISCUSSION THREAD

**Discussion by vladmdgolam** [yuanqing/create-figma-plugin#150] [Q&A]
Hello!

I'm curious if there's an option to change the output path of manifest.json during the build process. I'd like to share my plugin with colleagues while it's in development by providing them with a zipped folder containing the plugin's build. Has anyone encountered this issue before, and if so, how can it be done?

**Comment by fwextensions** [comment]
I'd like the same thing, and there seems to be an issue for it: https://github.com/yuanqing/create-figma-plugin/issues/157
